e2e/runner/evm.go

```diff
@@ -47,7 +47,7 @@ func (r *E2ERunner) ETHDeposit(
 	}()
 	r.EVMAuth.Value = amount
 
-	tx, err := r.GatewayEVM.Deposit0(r.EVMAuth, receiver, revertOptions)
+	tx, err := r.GatewayEVM.Deposit1(r.EVMAuth, receiver, revertOptions)
```

Comment:
Just curious, the live `e2e` tests are running against `develop` branch. Once it's merged, will this `r.GatewayEVM.Deposit1` call be interacting the method we want in live network (running an old gateway)? Looks like the `Deposit1` on existing gateway is another method.